parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/ParallelEoSStreamProcessor.java

```diff
@@ -13,14 +14,15 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 import java.util.function.Function;
 
 import static io.confluent.parallelconsumer.internal.UserFunctions.carefullyRun;
 
 @Slf4j
-public class ParallelEoSStreamProcessor<K, V> extends AbstractParallelEoSStreamProcessor<K, V>
-        implements ParallelStreamProcessor<K, V> {
+public class ParallelEoSStreamProcessor<K, V> extends AbstractParallelEoSStreamProcessor<K, V> implements ParallelStreamProcessor<K, V> {
```

Comment:
hmm, .editorconfig setup must have an issue, as it's introduction extra whitespace changes here. Something must not be quite sync'd with the default IDEA formatting.